_data/ru/intro.yml

```diff
@@ -1,11 +1,11 @@
 TC39:
   title: TC39
-  desc: Ecma International TC39 - это группа, состоящая из разработчиков JavaScript, имплементоров, ученых и других участников, сотрудничающих с сообществом для поддержки и развития JavaScript.
+  desc: Ecma International TC39 — это группа, состоящая из разработчиков JavaScript, имплементаторов, исследователей и других участников, сотрудничающих с сообществом для поддержки и развития JavaScript.
   ecma-logo: Мы являемся частью <br/> <a class="ecma-logo-box" href="https://www.ecma-international.org/"> <img src="assets/img/ecma-logo.svg" alt="Ecma International" /></a>
 contribute:
-  title: Сделать вклад
-  desc: TC39 приветствует участие сообщества JavaScript, будь то обратная связь по существующим предложениям, улучшения документации, тестирование, имплементации или даже идеи, связанные с особенностями языка. Подробности смотрите в нашем <a href="https://github.com/tc39/ecma262/blob/master/CONTRIBUTING.md">руководстве для желающих сделать вклад</a>.
+  title: Принять участие
+  desc: "TC39 приветствует любую помощь от сообщества JavaScript: отзывы и мнения о существующих предложениях, улучшение документации, тестирование или имплементации, или даже идеи для новых возможностей языка. Подробности смотрите в нашем <a href="https://github.com/tc39/ecma262/blob/master/CONTRIBUTING.md">руководстве для желающих принять участие в проекте</a>."
```

Comment:
I think build is failing because quotes in `href` value should be escaped.

Reply:
Yep, already fixed.